src/docfx/config/Config.cs

```diff
@@ -15,6 +15,16 @@ internal class Config
         private static readonly string[] s_defaultContentInclude = new[] { "docs/**/*.{md,yml,json}" };
         private static readonly string[] s_defaultContentExclude = Array.Empty<string>();
 
+        /// <summary>
+        /// Docset name.
+        /// </summary>
+        public readonly string DocsetName = string.Empty;
```

Comment:
We can have just a single config `Name` and use `{Product}.{DocsetName}` convension